assets/sass/config/_variables.scss

```diff
@@ -329,3 +327,26 @@ $z-index-sk-side-sheet: 10000;
 $z-index-sk-side-sheet-overlay: 9990;
 $z-index-sk-header: 9980;
 $z-index-sk-overlay-notification: 9970;
+
+// Start Authorize Application screen specific styles.
+$c-authorize-application-background: $c-base;
+$c-authorize-application-sys-light-primary: #0b57d0;
```

Comment:
Colour variables have been named according to their identified token names where possible e.g.:

![image](https://github.com/google/site-kit-wp/assets/18395600/842c97c2-b2db-4dec-84c6-99a593c24848)

Reply:
Nice one!